src/Sylius/Behat/Context/Ui/CheckoutContext.php

```diff
@@ -428,6 +447,22 @@ public function iProceedSelectingPaymentMethod($paymentMethodName = 'Offline')
     }
 
     /**
+     * @When I do not modify shipping method
+     */
+    public function iDoNotModifyShippingMethod()
+    {
+        $this->selectShippingPage->selectShippingMethod(null);
```

Comment:
I think we shouldn't even set `null` in this step. Can't we leave this method empty?